### ISSUE TITLE
Input component causing accessibility violation

### ISSUE THREAD

**Issue by codycaldwell** [react-toolbox/react-toolbox#1887]
The Input component defines an input tag along with a role attribute equal to 'input'. This causes some accessibility scanners to raise the following violation: 

_"The WAI-ARIA role(s) and/or attribute(s) input are not valid for the element input"_

Is there a way we can remove the redundant role or at least provide a way to override the value?

**Comment by theajr** [comment]
@rubenmoya

As per [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role), we don't need to have role for semantic elements i.e. input and textarea etc. 

role should be **textbox** instead of **input**

> Semantic elements are more concise and require no JavaScript to support textbox features.

**Comment by CwPranay** [comment]
Hi, I’d like to work on this issue. Can I take it?
